rpc/server.gen.go

```diff
@@ -0,0 +1,201 @@
+// Package rpc provides primitives to interact with the openapi HTTP API.
```

Comment:
Can we avoid comitting generated files to main, similar to how we do with protobufs?

Ditto for the other `.gen.go` files.

Reply:
Yes, there was an issue related to the imports inside generated files giving headaches to `go mod tidy` and `go mod vendor` but I fixed that with the `rpx/rpc.go` file